### PR TITLE
Configure language configuration for css, html, json, javascript, typescript, yaml.

### PR DESCRIPTION
**Pull request by angelozerr**
This PR configures language configuration for each bluesky languages.  language configuration is managed by tm4e and it supports for the moment `auto close` & `on enter` (ex : generate a new indentation after a bracket but not after some other pattern when enter is done).

 * here a little demo with JSON which doesn't use language configuration

jsonwithoutlanguageconfiguration

 * here a little demo with JSON with  language configuration:

jsonwithlanguageconfiguration

Signed-off-by: angelozerr <angelo.zerr@gmail.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
@mickaelistria could you review this PR please or tell me feedback if you prefer not using it. Thanks!

**Comment by angelozerr** [comment]
Thanks! I have the intention to implement matching bracket, but https://bugs.eclipse.org/bugs/show_bug.cgi?id=521031 should be implemented.

If you accept it, I will do a PR for aCute too.

File: org.eclipse.bluesky/build.properties [hunk 1]
```diff
@@ -3,5 +3,5 @@ output.. = bin/
 bin.includes = META-INF/,\
                .,\
                plugin.xml,\
-               grammars/,\
-               language-servers/
+               language-servers/,\
+               language-clients/
```
Comment (mickaelistria) [inline org.eclipse.bluesky/build.properties]: Can we call it `language-configurations` to better map to the goal and content of this folder?

File: org.eclipse.bluesky/language-clients/css/language-configuration.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (mickaelistria) [inline org.eclipse.bluesky/language-clients/css/language-configuration.json]: On all those, files, I believe it is important to place a comment or other kind of annotation to mention that those are taken from VSCode.

Comment (angelozerr) [reply org.eclipse.bluesky/language-clients/css/language-configuration.json]: It's a JSON file, how can we write comments?

File: org.eclipse.bluesky/build.properties (hunk 1 above)
Comment (angelozerr) [reply org.eclipse.bluesky/build.properties]: done

File: org.eclipse.bluesky/language-clients/css/language-configuration.json (hunk 2 above)
Comment (angelozerr) [reply org.eclipse.bluesky/language-clients/css/language-configuration.json]: Forget my comment, GSON is able to parse comments even if comment is not in the JSON specification. I have added license, etc

**Comment by angelozerr** [comment]
@mickaelistria I have commented JSON files + uses language-configurations as folder name. Is there other thing that you wish I change?

**Comment by mickaelistria** [comment]
Thanks!

**Pull request merged**